pkg/util/encryptionconfig/config_test.go

```diff
@@ -0,0 +1,151 @@
+package encryptionconfig
+
+import (
+	"context"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apiserver/pkg/storage/value/encrypt/identity"
+)
+
+var serviceAccountGVR = schema.GroupVersionResource{
+	Group:    "",
+	Version:  "v1",
+	Resource: "serviceaccounts",
+}
+
+var deploymentGVR = schema.GroupVersionResource{
+	Group:    "apps",
+	Version:  "v1",
+	Resource: "deployments",
+}
+
+func TestGetEncryptionTransformersFromSecret_Basic(t *testing.T) {
+	encryptionConfigFilepath := filepath.Join("testdata", "encryption-provider-config-wildcard.yaml")
+	configBytes, _ := os.ReadFile(encryptionConfigFilepath)
```

Comment:
Instead of the `filepath` + `os.Read` can use:
 https://github.com/rancher/backup-restore-operator/blob/155b1a97b134e43fdb88130c898f0d72690365a1/e2e/test/test.go#L14
 
 this way the tests aren't dependent on specific paths
 
 can also probably put move that into a `pkg/test` instead of `e2e/test` if you want

Reply:
Maybe as a follow up PR we move that to test, and consider making it more generic? To use:
```
func Data(testFs embed.FS, filename string) []byte {
```

So then we can still keep organizing each pkgs test stubs in place?